charts/base-cluster/templates/ingress/nginx.yaml

```diff
@@ -26,6 +26,7 @@ spec:
         use-proxy-protocol: "true"
         use-gzip: "true"
         enable-brotli: "true"
+        enable-underscores-in-headers: true
```

Comment:
Did you make sure that this works as a boolean? The other flags are strings

I'd rather have a real boolean, but you should make sure it works and if it does change the other flags to booleans as well.

Reply:
It is a bit confusing, i would say real bool should be right.  
If you look into the configmap values: https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/
you can see, that for example brotli is also declared `bool`, but the value has double quotes. In contrast the underscores option is declared the same, but the value has no double quotes.  

So i guess it should be right?

Reply:
Just tested it, works without problems like this.